lib/step_flow/migrations/create_workflow_rights.ex

```diff
@@ -0,0 +1,20 @@
+defmodule StepFlow.Migration.CreateWorkfkowRights do
+  use Ecto.Migration
+  @moduledoc false
+  def change do
+
+    create table(:step_flow_workflow_rights) do
+      add(:right, :string)
+      add(:group, :string, default: "")
+
+      add(
+        :workflow_definition_id,
+        references(:step_flow_workflow_definition, on_delete: :nothing)
+      )
```

Comment:
don't had this field here.
The WorkflowDefinition requires to have a `has_many` relation.

Like that the Workflow can have also some rights ...